src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {string, node, oneOf} from 'prop-types';
+import style from './Thumbnail.st.css';
+import Check from 'wix-ui-icons-common/Check';
+import {Thumbnail as CoreThumbnail, ThumbnailProps as CoreThumbnailProps} from 'wix-ui-core/Thumbnail';
+import {withStylable} from 'wix-ui-core/withStylable';
+import {Text} from '../Text';
+
+export interface ThumbnailProps extends CoreThumbnailProps {
+  /** image of the thumbnail */
+  image?: React.ReactElement<any>;
+  /** title of the thumbnail */
+  title: string;
+  /** description of the thumbnail */
+  description?: string;
+}
+
+const StyledThumbnail = withStylable<CoreThumbnailProps>(
+  CoreThumbnail,
+  style,
+  () => ({})
+);
+
+export const Thumbnail: React.SFC<ThumbnailProps> = props => {
+  const {children, title, image, description, ...rest} = props;
+
+  return (
+      <StyledThumbnail {...rest} selectedIcon={<Check size="24"/>}>
+        <div className={style.container}>
+          {image && <div data-hook="image">{image}</div>}
+          <Text className={style.title} bold>{title}</Text>
+          {description && <Text className={style.description}>{description}</Text>}
+        </div>
+      </StyledThumbnail>
+  );
+};
+
+Thumbnail.displayName = 'Thumbnail';
```

Comment:
Why do we need this?
it is inferred from the function/ class name.
https://reactjs.org/docs/react-component.html#displayname

Reply:
My only concern is whether the autodoc needs it explicitly. @argshook

Reply:
`react-docgen-typescript` currently has a bug, it doesn't consider `displayName` at all and instead relies on class name or fallsback to filename. since this is a stateless component, it will display filename. however, this bug is on hunt and should be resolved, thus i would say let's keep explicit `displayName`, it's the proper way